### ISSUE TITLE
Support for PHPUnit 8.0

### ISSUE THREAD

**Issue by osteel** [cloudcreativity/json-api-testing#7]
Hi,

I'm trying out [laravel-json-api](https://github.com/cloudcreativity/laravel-json-api) at the moment and it looks very promising.

The only thing that bugged me a little so far is that I had to downgrade PHPUnit from 8.0 back to 7.0 (Laravel 5.8 optionally allows to [upgrade to PHPUnit 8.0](https://laravel.com/docs/5.8/upgrade)) to be able to use the test helpers.

I wonder how much of a pain it'd be for this package to also support PHPUnit 8.0? (I can see it supports both 6.0 and 7.0 at the moment).

The 8.0 changelog can be found [here](https://github.com/sebastianbergmann/phpunit/blob/8.0.0/ChangeLog-8.0.md#800---2019-02-01).

Thanks!

**Comment by lindyhopchris** [comment]
Hi! Good point... I think it should be ok... I'll take a look at it today to see how easy it would be.

**Comment by lindyhopchris** [comment]
@osteel could you give this a go:

```bash
$ composer require --dev cloudcreativity/laravel-json-api:dev-phpunit8
```

**Comment by osteel** [comment]
@lindyhopchris hey mate, thanks for the quick response.

Just did that - updated the dependency and upgraded back PHPUnit from 7.0 to 8.0, and wrote a basic `doRead()->assertRead()` test for one of the resources - all went well.

So from here it seems like PHPUnit 8.0 could already be supported, although I don't know if this basic test is enough to say this with confidence.

**Comment by lindyhopchris** [comment]
Well it hasn't broken any of the apps we've got on PHPUnit 7, so I've tagged and released as `v1.1.0`. If you encounter any problems, just open a new issue describing the problem!

**Comment by osteel** [comment]
@lindyhopchris awesome! Thanks again for your reactivity 👍
